android/libraries/rib-badoo/src/main/java/com/badoo/common/rib/routing/action/MultiRoutingAction.kt

```diff
@@ -0,0 +1,33 @@
+package com.badoo.common.rib.routing.action
+
+import com.badoo.common.rib.BaseViewRouter
+import com.uber.rib.core.RibView
+
+class MultiRoutingAction< V : RibView>(
```

Comment:
`Multi` prefix is too vague, better find more suitable name
I would call it `Composite` as a disposable